packages/graphql_server/src/api/githubApi.ts

```diff
@@ -0,0 +1,94 @@
+import axios, { AxiosResponse } from 'axios'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+type ProjectFounder = {
+  name: string
+  login: string
+  twitterUsername: string
+}
+
+type GitHubCommitHistory = {
+  defaultBranchRef: {
+    target: {
+      history: {
+        edges: {
+          node: {
+            author: {
+              user: {
+                name: string
+                login: string
+                twitterUsername: string
+              }
+            }
+          }
+        }[]
+      }
+    }
+  }
+}
+
+/**
+ * Returns a Array of Founders with their names, login names and twitter handles. This method goes trough the commit history of a specific repo
+ * and fetches teh first 5 commits, which are most likley the initiators of a project. It then removes duplicates, because several commits can be from the
+ * same person, but shouldn't be returned within the Array
+ * @param owner: name of the owner of the github repo
+ * @param name: name of the github repo
+ * @returns An Array of the project founders
+ */
+export async function getRepoFounders(owner: string, name: string): Promise<ProjectFounder[]> {
```

Comment:
When an invalid values like an empty string is passed an exception is thrown. Please fix this and let it display a suitable message on console.error instead f.e. "Founders of <reponame> owned by <ownername> could not be found"